universal-login-contracts/lib/estimateGas.ts

```diff
@@ -14,13 +14,13 @@ export const calculateFinalGasLimit = (gasCall: utils.BigNumberish, gasBase: uti
 export const calculatePaymentOptions = (msg: SignedMessagePaymentOptions) =>
   ({gasLimit: calculateFinalGasLimit(msg.gasCall, msg.gasBase)});
 
-export const calculateGasBase = (message: Omit<Message, 'gasLimit'>) => {
+export const calculateGasBase = (message: Omit<Message, 'gasLimit'>, networkVersion: ChainVersion, walletVersion: WalletVersion) => {
```

Comment:
To consider next PR: join `networkVersion` and `walletVersion` into `gasCalculationEnvironment`?